core/src/test/scala/org/apache/spark/rdd/PairRDDFunctionsSuite.scala

```diff
@@ -614,7 +614,7 @@ class PairRDDFunctionsSuite extends SparkFunSuite with SharedSparkContext {
     val e = intercept[SparkException] {
       pairs.saveAsNewAPIHadoopFile[NewFakeFormatWithCallback]("ignored")
     }
-    assert(e.getCause.getMessage contains "failed to write")
+    assert(e.getCause.getMessage contains "Task failed while writing rows")
```

Comment:
how it happens that you have to change this?

Reply:
That is also my question. I believe this error message should not been affected by current change, but CI keep complaining about this.

So I modified it for testing purposes to see if this would really change the response of CI.